universal-application-tool-0.0.1/app/services/applicant/ApplicantQuestion.java

```diff
@@ -90,6 +90,10 @@ public TextQuestion getTextQuestion() {
     return new TextQuestion();
   }
 
+  public MultiSelectQuestion getMultiSelectQuestion() {
```

Comment:
nit: 'get' here may imply it's a singleton or the same thing each time it's called, recommend renaming to `make` or `create`

Reply:
Ok! I'll call it `createMultiSelectQuestion` - I'm working on the refactor of `ApplicantQuestion` after this so it will be a reminder to name the other methods too